config/api.yml

```diff
@@ -1932,6 +1932,8 @@
         :identifier: service_admin
       - :name: delete
         :identifier: service_delete
+      - :name: add_resource
+        :identifier: service_edit
```

Comment:
@mkanoor can you verify this product feature used? I didn't see one for assigning resources